ansible/configs/ocp4-workshop/requirements.yml

```diff
@@ -4,3 +4,7 @@
 - src: https://github.com/redhat-gpte-devopsautomation/ftl-injector
   name: ftl-injector
   version: v0.16.0
+
+# From 'Stouts.wsgi'
+- src: https://github.com/Stouts/Stouts.wsgi
```

Comment:
Used by status report?

Reply:
@jimrigsbee Do I need to move this to the status reporting API PR? https://github.com/RedHatTraining/agnosticd/pull/8/files

Reply:
Jim says "yes" in email.